dev/tools/lib/globals.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
uber-nit: constants?

Reply:
> roll dev was the only existing functionality, correct?

correct

Reply:
added the function `stdoutToString` here, is that cool?

Reply:
SGTM

Reply:
I just have an irrational fear of globals